docs/tutorials/multimodal/multimodal-faq.md

```diff
@@ -53,3 +53,21 @@ We used this script in our examples about [Petfinder Pawpularity](https://github
 and [Feedback Prize - Predicting Effective Arguments](https://github.com/autogluon/autogluon/tree/master/examples/automm/kaggle_feedback_prize). 
 You may refer to these examples for more details.
 
+## How does AutoGluon MultiModal handle multiple images per sample?
+
+We provide two options:
+1. Utilizing multiple image columns in a dataframe.
```

Comment:
Do we want to point user to how to specify multiple image columns, or they will be auto detected?

Reply:
Thanks. Added one sentence of introducing the auto-detection.